x-pack/plugins/observability/public/pages/alerts/alerts_flyout/index.tsx

```diff
@@ -70,9 +79,19 @@ export function AlertsFlyout({
     return (alerts ?? []).map(parseObservabilityAlert);
   }, [alerts, observabilityRuleTypeRegistry]);
 
+  const onSelectPage = useCallback(
+    (activePage: number) => {
+      onSelectedAlertIndexChange?.(activePage);
+    },
+    [onSelectedAlertIndexChange]
+  );
+
   let alertData = alert;
   if (!alertData) {
-    alertData = decoratedAlerts?.find((a) => a.fields[ALERT_UUID] === selectedAlertId);
+    if (selectedAlertId != null) {
```

Comment:
Sorry for jumping between lines :) 
So, this if-case is also useless?

Reply:
Possibly! The selectedAlertId is only passed by this storybook example:

https://github.com/elastic/kibana/blob/0f68fcf5af75adc608e7d189c680dd9a0eea2a7f/x-pack/plugins/observability/public/pages/alerts/alerts_flyout/alerts_flyout.stories.tsx#L58:L72

Well spotted! Looks like a good candidate for a cleanup in a later PR ⚡